src/components/molecules/alert-information.tsx

```diff
@@ -5,35 +5,42 @@ import {useNavigation} from '@react-navigation/native';
 
 import {colors, text} from 'theme';
 import {Card} from 'components/atoms/card';
-import {StateIcons} from 'assets/icons';
+import {StateIcons, AppIcons} from 'assets/icons';
 import {ScreenNames} from 'navigation';
 
-export const AlertInformation = forwardRef<TouchableWithoutFeedback>(
-  (props, ref) => {
-    const {t} = useTranslation();
-    const navigation = useNavigation();
+interface AlertInformationProps {
+  bluetoothOff?: boolean;
+}
 
-    return (
-      <Card
-        ref={ref}
-        padding={{h: 10, r: 16}}
-        onPress={() => navigation.navigate(ScreenNames.MyCovidAlerts)}
-        type="info"
-        arrowColor={colors.black}>
-        <View style={styles.row}>
+export const AlertInformation = forwardRef<
+  TouchableWithoutFeedback,
+  AlertInformationProps
+>(({bluetoothOff}, ref) => {
+  const {t} = useTranslation();
+  const navigation = useNavigation();
+
+  return (
+    <Card
+      ref={ref}
+      padding={{h: 10, r: 16}}
+      onPress={() => navigation.navigate(ScreenNames.MyCovidAlerts)}
+      type="info"
+      arrowColor={colors.black}>
+      <View style={styles.row}>
+        {bluetoothOff ? (
+          <AppIcons.BluetoothOff width={40} height={30} color={colors.text} />
```

Comment:
40x30 vs 40x40 (just checking as viewBox is 0 0 23 30)? Maybe ask @helloantoine for the correct size in both cases (if you didn't already)

Reply:
@helloantoine can you please confirm the size for this icon?